service/main.go

```diff
@@ -45,6 +46,7 @@ func main() {
 		panic(fmt.Sprintf("Unable to load config %s", err))
 	}
 	level.Info(logger).Log("message", fmt.Sprintf("loading config '%s' completed", env.ConfigFilePath))
+	validations.SetImageURIs(config.ImageURIs)
```

Comment:
Would be nice if there were a known and shared config flowing through the system to work with. As other things are added (like a git domain) this will feel dirtier than it already does.

Reply:
What about setting this in `loadConfig`? Seems like it might be a better fit.

Reply:
💯 good call.